apps/demo/app/routes/index.tsx

```diff
@@ -48,339 +50,377 @@ export default function Index() {
+              title: 'See designs on Figma',
+              href: 'https://www.figma.com/file/zpDGiKGaY35SEnfKB2uzeZ/Radius-Demo-Site?type=design&t=GqzqcLwssKZzV8o7-0',
+              icon: Figma as IconType,
+            },
+            {
+              'aria-label': mode
+                ? 'Switch to dark mode'
+                : 'Switch to light mode',
+              title: mode ? 'Switch to dark mode' : 'Switch to light mode',
+              as: 'button',
+              onClick: toggleMode,
+              icon: (mode ? DarkMode : LightMode) as IconType,
+            },
+          ]}
+          logos={
+            <>
+              <a
+                href="#"
+                data-radius-watch-token-changes
+                data-radius-target-selector="img"
+                data-radius-target-attribute="src"
+                data-radius-replace-value="/brand/{--brand}/assets/primary-logo-{--mode}.svg"
+              >
+                <RadiusAutoLayout
+                  as="img"
+                  alt="Company logo"
+                  src="/brand/saddles/assets/primary-logo-light-mode.svg"
+                  style={{
+                    height: `var(${radiusTokens.component.sizing.navigation.primaryLogo})`,
+                  }}
+                />
+              </a>
+            </>
           }
-          ctaUrl="/posts"
-          imageAlignment="right"
-          data-radius-watch-token-changes
-          data-radius-target-selector="img"
-          data-radius-target-attribute="src"
-          data-radius-replace-value="/brand/{--brand}/assets/semantic.image.hero.headerImage.webp"
-          className={`demo-hero ${css`
-            background: url(/brand/saddles/assets/semantic.image.hero.backgroundImage.webp);
-          `}`}
+          navItems={[
+            {
+              href: '#',
+              label: 'Menu Item 1',
+              selected: true,
+            },
+            {
+              href: '#',
+              label: 'Menu Item 2',
+            },
+            {
+              href: '#',
+              label: 'Menu Item 3',
+            },
+          ]}
+          socials={[
+            {
+              icon: Twitter as IconType,
+              href: '#',
+              'aria-label': 'Descriptive text',
+            },
+            {
+              icon: LinkedIn as IconType,
+              href: '#',
+              'aria-label': 'Descriptive text',
+            },
+            {
+              icon: Instagram as IconType,
+              href: '#',
+              'aria-label': 'Descriptive text',
+            },
+            {
+              icon: Youtube as IconType,
+              href: '#',
+              'aria-label': 'Descriptive text',
+            },
+          ]}
+          className={css`
+            margin: 0 auto;
+            max-width: ${pageMaxWidth}px;
+          `}
+          // TODO: shadow is now a page-level concern since we are adding wrappers
+          // hasShadow={isScrolled}
```

Comment:
Previously, shadow had been integrated directly into the Nav component. But now that the nav only takes up a portion of the screen width, it should be removed and applied directly to the wrapper